stdlib/format.mli

```diff
@@ -918,6 +925,18 @@ val get_formatter_out_functions : unit -> formatter_out_functions
   @since 4.01
 *)
 
+val utf8_scalar_width: string -> pos:int -> len:int -> int
+(** [utf8_scalar_width s ~pos ~len] is the number of unicode scalar values in
+    the substring [String.sub s pos len]. Invalid byte sequences are implictly
+    replaced by [U+FFFD] since this yields a better width approximation for
+    other ascii-based encoding scheme like ISO-8859-15. This is the default
+    [out_width] function.
```

Comment:
I would say "default [out_wdith] function since 5.4.", given that previous version did not have `out_width` but behaved as if they used `ascii_width`.